### ISSUE TITLE
tx parse error for MsgUnlockPeriodLock and MsgCreatePool

### ISSUE THREAD

**Issue by phswordfish** [osmosis-labs/osmosis#1079]
When using v7.0.3 archive node and client `osmosisd`, some older transaction cannot be queried. For examples:

`osmosisd q tx 9E26F0602680744698D714B1CE8AC02240AB0BF96266880E07C308C2673BE36F
Error: unable to resolve type URL /osmosis.lockup.MsgUnlockPeriodLock: tx parse error`

`osmosisd q tx D93C8501DD632E02A61B13691CC8C5E7E72247068B79C5A128F7A7AD6346ECC6
Error: unable to resolve type URL /osmosis.gamm.v1beta1.MsgCreatePool: tx parse error`

`osmosisd query txs --height 10973 --events 'tx.height=10973'
Error: unable to resolve type URL /osmosis.gamm.v1beta1.MsgCreatePool: tx parse error`

I will try using v6.4.0 osmosisd binary to query such v7.0.3 node and update the ticket here.

**Comment by ValarDragon** [comment]
yeah, we gotta figure this out. Going to close in favor of #1017 since that has more context for the issue
